pgmpy/estimators/PC.py

```diff
@@ -258,6 +260,11 @@ def build_skeleton(
                     f"ci_test must either be one of {list(CI_TESTS.keys())}, or a function. Got: {ci_test}"
                 )
 
+        if expert_knowledge is not None:
```

Comment:
I would suggest moving this to `ExpertKnowledge.__init__` method.

Reply:
@Nimish-4 Sorry, I had missed that this is for the case when expert_knowledge is None. But even for that case, I would suggest initializing an expert_knowledge object with default values at the start of the `estimate` method. That would help us avoid doing such ad hoc checks.